ReactAndroid/src/main/java/com/facebook/react/uimanager/BaseViewManager.java

```diff
@@ -287,7 +287,10 @@ public void setViewState(@NonNull T view, @Nullable ReadableMap accessibilitySta
       view.setSelected(false);
     }
     view.setTag(R.id.accessibility_state, accessibilityState);
-    view.setEnabled(true);
+    
+    if (!accessibilityState.hasKey("disabled")) {
```

Comment:
I think this condition will be triggered for "disabled: false" as well. Is this intentional, vs checking with `getBoolean()`?

Reply:
You are right I've also added a check with `getBoolean()`.